<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Manejo de errores 3</title>
</head>
<body>
    <script>
        //ASYNC/AWAIT
        async function sumarNumeros(a, b) {
            if (typeof a != 'number' || typeof b != 'number'){
                throw new Error('Alguno de los argumentos no es un numero');
                
            }return a + b
        }

        async function manejarErrores() {
            try {
                let resultado = await sumarNumeros('2', 3);
                console.log(resultado);
            }
                catch (error){
                    console.error(error.message);
                }
        }

        manejarErrores();
    </script>
</body>
</html>